### DISCUSSION TITLE
Zitadel console - error management

### DISCUSSION THREAD

**Discussion by santhosh-stitch** [zitadel/zitadel#6659] [Q&A]
<img width="852" alt="console-ui" src="https://github.com/zitadel/zitadel/assets/104059308/548cfeb5-2431-49c0-9ed8-f0689b2a1f9f">

When the Zitadel self-host database was down, the information above was displayed in the Zitadel console.
It displays connection information.

Is it possible to modify the Zitadel console's error page?

**Comment by adlerhurst** [comment]
Thats a mistake. We will fix it and hide the information asap
